api/server.py

```diff
@@ -141,6 +146,31 @@ def get_all_companies():
     return jsonify({'All Companies' : output})
 
 
+@app.route('/api/projects/delete', methods=['DELETE'])
+def delete_project():
+    projects = mongo.db.projects
+
+    project_name = request.json['project_name']
```

Comment:
Front-end will have the _id of the project entry, so we should use that instead of project_name in case there are duplicate projects with the same name

Reply:
In that case, I'll edit the rest of the server file to work based off of project_id instead, is that alright?